readme.txt

```diff
@@ -56,7 +56,7 @@ Please help us improve the plugin by enabling [anonymous statistics and error co
 * 2017-01-17
 * Fix: Can't back-fill more than one order per user
 * Fix: Transaction details sent multiple times
-* Fix: Correct the mapping of WooCommerce order status to Sift Science transaction status
+* Fix: Correct the mapping of WooCommerce order status to Sift  transaction status
```

Comment:
In the past the name was Sift Science. Let's keep the changelog messages using the old name.